src/pages/components/BrowserContextMenu.qml

```diff
@@ -28,6 +29,11 @@ Rectangle {
     readonly property bool active: visible
     readonly property bool landscape: width > height
 
+    readonly property string url: !isJavascriptFunction ? linkHref : ""
+    readonly property bool isJavascriptFunction: linkProtocol === "javascript"
+    readonly property bool isLink: !isJavascriptFunction && linkHref.length > 0 && imageSrc.length === 0
```

Comment:
My proposal:

mailto:

1. Write an email (anything better?)

2. Share

3. Copy to clipboard

sms:

1. Send a message

2. Share

3. Copy to clipboard

tel:

1. Call

2. Share

3. Copy to clipboard

geo:

1. Open

2. Share

3. Copy to clipboard

=> Conceptually all these four would have same context menus but open text would be context sensitive. I'd not start customizing "Copy to clipboard" behaviour as there can be also a body in query arguments. This would already be better than Google Chrome has they are showing for tel/sms/geo/mailto

- "Open in new tab"

- "Open in incognito tab"

- "Copy link address"

- "Copy link text"

The mailto protocol does an exception => that has "Copy email address" that tries to extract email address but fails on CC and BCC only cases (TO is not given).